core/chain-configs/src/genesis_config.rs

```diff
@@ -3,11 +3,13 @@
 //! NOTE: chain-configs is not the best place for `GenesisConfig` since it
 //! contains `RuntimeConfig`, but we keep it here for now until we figure
 //! out the better place.
+use anyhow::Context;
 use std::fs::File;
 use std::io::{BufReader, Read};
 use std::path::{Path, PathBuf};
 use std::{fmt, io};
 
+use anyhow::Result;
```

Comment:
To be honest, I don't like overloading commonly used types (Result, Option, etc), so I would consider explicitly stating that the return value is `anyhow::Result<...>`

Reply:
@matklad What is the recommended codestyle for using `Result`? `anyhow::Result` or `Result`?

Reply:
@frol Yes, it feels weird. I tried using `anyhow::Result`, but I reversed it, because I stopped using regular `Result<>` completely. Though, I think `anyhow::Result` should be the default, the std `Result`, is hard to use, so we ended up using `.unwrap()` everything in the code, which is much worse.